cypress/support/generateArticle.js

```diff
@@ -0,0 +1,10 @@
+import { faker } from '@faker-js/faker';
+
+export function generateArticle () {
+  return {
+    title: faker.lorem.word(5), // Генерація випадкової назви статті
```

Comment:
The `faker.lorem.word(5)` method is used to generate a title. Note that `faker.lorem.word()` typically generates a single word. If you intend to generate a title with multiple words, consider using `faker.lorem.words(5)` instead.